src/main/java/net/sky/pieces/PieceType.java

```diff
@@ -0,0 +1,15 @@
+package net.sky.pieces;
+
+public enum PieceType {
+    PAWN('p'), KNIGHT('n'), ROOK('r'), BISHOP('b'), QUEEN('q'), KING('k');
```

Comment:
인스턴스 변수를 갖는 `enum` 의 경우 한 줄에 하나씩 선언해주세요.
```suggestion
    PAWN('p'),
    KNIGHT('n'),
    ROOK('r'),
    BISHOP('b'),
    QUEEN('q'),
    KING('k');

Reply:
이런 부분 짚어주시는거 너무 감사합니다! 아는게 많이 없다보니까 이런 세심한 피드백이 너무 좋은 것 같아요!